Hardware/index.md

```diff
@@ -4,23 +4,47 @@ SPDX-FileCopyrightText: 2020 seL4 Project a Series of LF Projects, LLC.
 ---
 # Supported Platforms
 
-## Verification status
+### Summary
 
-The seL4 proofs are only for specific platforms, as noted in the *Status* column as follows:
+seL4 is available on 3 major hardware architectures, [ARM](#arm),
+[RISC-V](#risc-v) and [x86](#x86), for a number of platforms (with varying
+hardware features).
+
+seL4 is formally verified (to various degrees) for a subset of these
+configurations, i.e. combination of architecture, platform and hardware
+features.
```

Comment:
The "theses" equates platforms and configurations, which is not quite right and could be confusing. It's trying to explain with the side sentence, but a configuration is more specific than a arch/platform/hw feature (i.e. for each platform, you can still configure seL4 to have certain software features or not and configure various parameters). Not sure if we should go into detail on that here. We could just say

> "seL4 is formally verified for specific configurations for a subset of these platforms. The depth of the proofs and which properties are verified depend on the platform. The table below provides more details."

(then need to remove the next para)